stripe/test/resources/test_stripe_object.py

```diff
@@ -142,7 +142,13 @@ def test_pickling(self):
             'foo', 'bar', myparam=5)
 
         obj['object'] = 'boo'
-        obj.refresh_from({'fala': 'lalala'}, api_key='bar', partial=True)
+        obj.refresh_from(
+            {
+                'fala': 'lalala',
+                'emptystring': '',
```

Comment:
Can we put a comment here explaining that `emptystring` is trying to uncover a specific Python 3 unpickling issue?